### DISCUSSION TITLE
Switching languages

### DISCUSSION THREAD

**Discussion by RatedYK** [protomaps/PMTiles#359] [Q&A] [answered]
Hey, I can see that my .pmtiles file has languages (`name:${language}`) when looking at its attributes in the PMTiles Viewer. But i'm unsure of how to access these values or switch the language of the map.

Currently I'm rendering the tiles using maplibre-gl and tried following their docs here: https://maplibre.org/maplibre-gl-js/docs/examples/language-switch/ This however does not seem to work (map stops being rendered).

Any help is much appreciated, thanks!

**Comment by bdon** [comment]
This is not related to the PMTiles format, only the tileset and style you're using. What tileset are you using?

**Reply by RatedYK** [reply]
Oh I didn't know that, still very new to the world of map tiles!

I followed the Getting Started [setup](https://docs.protomaps.com/guide/getting-started) , so I believe I'm using a tile set from OpenStreetMap. 

As for the style I copied the json file from here https://maps.protomaps.com/#map=0.56/0/0&theme=light&renderer=maplibregl&tiles=https://build.protomaps.com/20240209.pmtiles

**Reply by bdon** [reply, marked as answer]
replace `name` with `name:en` or whatever other langauge code you need.
